vendor/bat-native-ads/src/bat/ads/internal/account/account.cc

```diff
@@ -199,4 +202,15 @@ void Account::OnDidRetryRedeemingUnblindedPaymentTokens() {
   BLOG(1, "Retry redeeming unblinded payment tokens");
 }
 
+void Account::OnCaptchaRequiredToRefillUnblindedTokens(
+    const std::string& captcha_id) {
+  BLOG(1, "Captcha required to refill unblinded tokens");
+
+  const WalletInfo wallet = GetWallet();
+  const std::string wallet_id = wallet.id;
```

Comment:
`const std::string&`

Reply:
I just got rid of the temporary, seems kind of pointless anyway.